runtime-sdk/src/dispatcher.rs

```diff
@@ -18,14 +18,15 @@ use oasis_core_runtime::{
         tags::Tags,
         types::TxnBatch,
     },
-    types::CheckTxResult,
+    types::{CheckTxMetadata, CheckTxResult, CheckTxWeight},
 };
 
 use crate::{
     context::{BatchContext, Context, RuntimeBatchContext},
     error::{Error as _, RuntimeError},
     module::{self, AuthHandler, BlockHandler, MethodHandler},
     modules,
+    modules::core::API,
```

Comment:
```suggestion
    modules::core::API as _,
```
(To avoid having a general "API" type imported.)